bin/updateSquadsCategory.ts

```diff
@@ -0,0 +1,54 @@
+import '../src/config';
+import createOrGetConnection from '../src/db';
+import { SquadSource } from '../src/entity';
+import { SourceCategory } from '../src/entity/sources/SourceCategory';
+import { categorizedSquads } from './categorizedSquads';
+import { In } from 'typeorm';
+import { runInQueueStream } from './common';
+
+interface Result {
+  title: string;
+  id: string;
+}
+
+(async () => {
+  const con = await createOrGetConnection();
+  await con.transaction(async (manager) => {
+    console.log('initializing stream');
+    const categories = await manager.getRepository(SourceCategory).find();
```

Comment:
Would `find({ title: In(categorizedSquads.map(item => item.category) })` would work here better to not query everything?

Reply:
We do this pull to ensure everything listed from their data was found and errors to be fixed. Thankfully we did, found some issues with their data and have been fixed now.

Reply:
We can add this condition on the actual update that we will run. Let me add it quickly.